File: app/src/main/java/com/textile/markeet/helpers/AppConstants.java
package com.textile.markeet.helpers;

import android.content.Context;

import java.util.regex.Pattern;

public class AppConstants {

    public static final String BASE_URL = "https://www.textilemarkeet.com/";
    public static Context mContext;

    public static final Pattern PASSWORD
            = Pattern.compile("^" +
            "(?=.*[0-9])" +
            "(?=.*[a-zA-Z])" +
            "(?=\\S+$)" +
            ".{6,}" +
            "$");

}

File: app/src/main/java/com/textile/markeet/helpers/AppConstants.kt
package com.textile.markeet.helpers

import android.content.Context
import java.util.regex.Pattern

class AppConstants {


    companion object {

        const val BASE_URL = "http://textilemarkeet.com/wp-json/textilemarkeet/v1/"
        var mContext: Context? = null
        const val TOKEN = "session_token"
        const val BEARER = "Bearer "

        val PASSWORD = Pattern.compile(
            "^" +
                    "(?=.*[0-9])" +
                    "(?=.*[a-zA-Z])" +
                    "(?=\\S+$)" +
                    ".{6,}" +
                    "$"
        )
    }
}